src/OrbitQt/main.cpp

```diff
@@ -161,6 +174,32 @@ void RunUiInstance(const DeploymentConfiguration& deployment_configuration,
   }
 }
 
+static QStringList ExtractCommandLineFlags(const std::vector<std::string>& command_line_args,
+                                           const std::vector<char*>& positional_args) {
+  QStringList command_line_flags;
+  absl::flat_hash_set<std::string> positional_arg_set(positional_args.begin(),
+                                                      positional_args.end());
+  for (std::string command_line_arg : command_line_args) {
+    if (!positional_arg_set.contains(command_line_arg)) {
+      command_line_flags << QString::fromStdString(command_line_arg);
+    }
+  }
+  return command_line_flags;
+}
+
+static std::vector<std::string> ExtractCaptureFilePaths(const std::vector<char*>& positional_args) {
```

Comment:
I am happy to discuss this and happy to be convinced otherwise: I would not do this filtering. 

I do not see any benefit to restrict the files that can be opened to .orbit files. But I see on downside: If the user tries to open a different file with Orbit (maybe even in windows explorer via "right-click > Open With..") then Orbit will just ignore this file. 

I think we should not filter for ".orbit", but if we do filter, we should at least display a warning to the user.

Reply:
I was thinking the same and then decided it's ok to leave it like this, but I was very much on the fence about this. Now that Anton also points it out, I'd agree :-)

Reply:
Thanks for pointing out this. I didn't notice the case of opening other format files with orbit.

Removed the filtering of file extension.